third_party/kourier-latest/kourier.yaml

```diff
@@ -297,7 +297,7 @@ spec:
         app: net-kourier-controller
     spec:
       containers:
-        - image: gcr.io/knative-nightly/knative.dev/net-kourier/cmd/kourier@sha256:cf8b6bc9195765272e37c86a9b70a23d264a39648f8e7335f2cd6ac25ce259f4
+        - image: gcr.io/gcp-compute-engine-223401/kourier-b74c3918b7eee585f87df62ccd297dc8:latest
```

Comment:
I'm not sure I understand why we have this diff

Reply:
net-kourier had an issue https://github.com/knative-sandbox/net-kourier/pull/813. The diff can be removed once the new image was pulled in this repo.